net/shorewall/files/vardir

```diff
@@ -0,0 +1,3 @@
+# move state dir out of ram
```

Comment:
Don't understand this comment... `/tmp` is a ramfs on OpenWRT.

Reply:
Removed